### DISCUSSION TITLE
set up docker-compose up -d ;error happed, How to fix it？ thanks

### DISCUSSION THREAD

**Discussion by Leelongqi** [getredash/redash#6475] [Q&A] [answered]
redash-worker-1     | [2023-09-23 15:54:23,420][PID:30][INFO][rq.worker] periodic: Job OK (59a84668e68687338646f735965213c58e814b32)
redash-worker-1     | [2023-09-23 15:54:23,420][PID:30][INFO][rq.worker] Result is kept for 600 seconds
redash-worker-1     | [2023-09-23 15:54:23,425][PID:7][INFO][rq.worker] periodic: 75540cdf868e7873f5eec072177423b13b98dece
redash-worker-1     | [2023-09-23 15:54:23,431][PID:31][INFO][rq.job.redash.tasks.queries.maintenance] job.func_name=redash.tasks.queries.maintenance.empty_schedules job.id=75540cdf868e7873f5eec072177423b13b98dece Deleting schedules of past scheduled queries...
redash-postgres-1   | 2023-09-23 15:54:23.447 UTC [31] ERROR:  relation "queries" does not exist at character 875
redash-postgres-1   | 2023-09-23 15:54:23.447 UTC [31] STATEMENT:  SELECT queries.query AS queries_query, queries.updated_at AS queries_updated_at, queries.created_at AS queries_created_at, queries.id AS queries_id, queries.version AS queries_version, queries.org_id AS queries_org_id, queries.data_source_id AS queries_data_source_id, queries.latest_query_data_id AS queries_latest_query_data_id, queries.name AS queries_name, queries.description AS queries_description, queries.query_hash AS queries_query_hash, queries.api_key AS queries_api_key, queries.user_id AS queries_user_id, queries.last_modified_by_id AS queries_last_modified_by_id, queries.is_archived AS queries_is_archived, queries.is_draft AS queries_is_draft, queries.schedule AS queries_schedule, queries.schedule_failures AS queries_schedule_failures, queries.options AS queries_options, queries.search_vector AS queries_search_vector, queries.tags AS queries_tags
redash-postgres-1   |   FROM queries
redash-postgres-1   |   WHERE queries.schedule IS NOT NULL ORDER BY queries.id
redash-worker-1     | [2023-09-23 15:54:23,451][PID:31][ERROR][rq.worker] Traceback (most recent call last):
redash-worker-1     |   File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context
redash-worker-1     |     self.dialect.do_execute(
redash-worker-1     |   File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute
redash-worker-1     |     cursor.execute(statement, parameters)
redash-worker-1     | psycopg2.errors.UndefinedTable: relation "queries" does not exist
redash-worker-1     | LINE 2: FROM queries
redash-worker-1     |              ^

**Comment by justinclift** [comment, marked as answer]
That's a strange error:

```
ERROR: relation "queries" does not exist at character 875
```

The `queries` table is part of Redash itself.  Is there any chance you forgot to initialise the database before starting the containers?

```
$ make create_database
$ make up
```

In a development setup, the database is wiped when you shut down the containers.  So, you need to initialise it each time you start them up.

**Reply by Leelongqi** [reply]
thanks, i have set up "docker-compose run --rm server create_db"  and then "docker-compose up -d", its ok~ thank you for your answer!
